src/js/components/orgs/cards/orgActions.tsx

```diff
@@ -118,7 +118,10 @@ const OrgActions = ({
       <Button
         label={i18n.t('Contribute Work')}
         variant="outline-brand"
-        className="slds-m-right_x-small tour-scratch-org-contribute"
+        className="slds-m-right_x-small
+          slds-m-left_none
+          slds-m-top_xx-small
```

Comment:
I don't think this top-margin gains us much, and it makes the btn and the dropdown no longer aligned.

![Screen Shot 2021-10-06 at 3 45 52 PM](https://user-images.githubusercontent.com/552316/136272741-32abd468-7c66-49b4-9dce-98f4b96bf637.png)

Reply:
Yeah, that was my assumption -- but even when stacked, I wasn't sure it was necessary. 🤷

Reply:
yeah, we have row-gap too so I can remove it. I think I was going back and forth there for a while.